de-lib/src/main/java/org/iplantc/de/resources/client/uiapps/widgets/AppsWidgetsContextualHelpMessages.properties

```
@@ -47,4 +47,9 @@ infoLabelHelp = This field accepts the following html tags: &lt;p&gt;, &lt;br&gt
 
 fileInputIsImplicit = <p>Selecting this option allows the app user to select input files without those files being included in the command line.</p>
 fileInputRepeatOptionFlag = <p>Selecting this option allows the "Argument option" defined for this parameter to be included on the command line before each filename selected by the user.</p>
-multiFilePrompt = Select multiple input files. Tip: You can also drag and drop files from the Data window.
\ No newline at end of file
+multiFilePrompt = Select multiple input files. Tip: You can also drag and drop files from the Data window.
+fileInputDefaultText = <p>The app will use this value by default. App will need to be updated if this file moved, deleted or renamed!</p>
+folderInputDefaultText = <p>The app will use this value by default. App will need to be updated if this folder moved, deleted or renamed!</p>
```

Just copying @ash3rz 's suggestion here too:
```suggestion
folderInputDefaultText = <p>The app will use this value by default. Note that the app will need to be updated if this folder is moved, deleted, or renamed.</p>
```